static/usage/v6/select/typeahead/angular/example_component_ts.md

```diff
@@ -0,0 +1,59 @@
+```ts
+import { Component, ViewChild } from '@angular/core';
+import { IonModal } from '@ionic/angular';
+import { Item } from './types'; 
+
+@Component({
+  selector: 'app-example',
+  templateUrl: 'example.component.html',
+})
+export class ExampleComponent {
+  @ViewChild('modal', { static: true }) modal!: IonModal;
+  
+  selectedFruitsText: string = '0 Items';
```

Comment:
~ In Typescript when you assign a default value of a primitive, it automatically infers/assigns the type, so the `: string` is redundant here. It wouldn't cause any issues and depending on linting rules it might throw a warning about it.